recipes/muse/build.sh

```diff
@@ -1,6 +1,16 @@
-#!/usr/bin/env bash
-
-make CPP="${CXX}" CPPFLAGS="${CXXFLAGS}"
+#!/bin/bash -euo
 
 mkdir -p "${PREFIX}/bin"
-cp MuSE "${PREFIX}/bin/"
+
+export INCLUDES="-I{PREFIX}/include"
+export LIBPATH="-L${PREFIX}/lib"
+export LDFLAGS="${LDFLAGS} -L${PREFIX}/lib"
+
+export CFLAGS="${CFLAGS} -O3"
+
+CC="${CC} -O3 ${LDFLAGS}" CFLAGS="${CFLAGS}" CXXFLAGS="${CXXFLAGS} -O3" \
+	CPP="${CPP}" CPPFLAGS="${CPPFLAGS} -I${PREFIX}/include" \
```

Comment:
```suggestion
	CPP="${CXX}" CPPFLAGS="${CXXFLAGS} -I${PREFIX}/include" \
```
By convention `CPP` is C pre-processor. 
For C++ compiler the usual name is `CXX`